x-pack/plugins/alerting/server/lib/last_run_status.ts

```diff
@@ -38,6 +38,7 @@ export const lastRunFromState = (
 
   if (warnings.length > 0) {
     outcome = RuleLastRunOutcomeValues[1];
+    outcomeMsg.push(...warnings);
```

Comment:
The changes to this file and its tests are pending feedback in:

If these changes are needed, these changes should be deleted from this PR and merged before this one from a separate PR which is already created: https://github.com/elastic/kibana/pull/148142

Reply:
Based on the discussion it was decided to keep the current behaviour.